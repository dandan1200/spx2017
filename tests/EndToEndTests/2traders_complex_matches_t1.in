SELL 0 GPU 5 35;
BUY 1 Router 4 22;
