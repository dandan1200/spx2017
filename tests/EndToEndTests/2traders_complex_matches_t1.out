MARKET OPEN;
ACCEPTED 0;
ACCEPTED 1;
MARKET BUY GPU 10 20;
MARKET BUY GPU 100 30;
MARKET SELL Router 5 20;
FILL 1 4;
MARKET BUY GPU 100 40;
FILL 0 5;
